runconfig/parse.go

```diff
@@ -441,6 +441,8 @@ func ParseRestartPolicy(policy string) (RestartPolicy, error) {
 			}
 
 			p.MaximumRetryCount = count
+		} else if len(parts) > 2 {
```

Comment:
@runcom I think the most case is `len(parts) == 2`,  the most case should be at the start. WDYT?:)